guide/onboarding/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
```

Comment:
Will you add a preview image later?

Reply:
Yeah I've updated this now, was waiting on some illustrations but I'll put something in for now and change later.